src/sentry/static/sentry/app/components/search/index.jsx

```diff
@@ -87,10 +87,11 @@ class Search extends React.Component {
     navigateTo(nextPath, router);
   };
 
-  saveQueryMetrics = debounce(
-    query => analytics(`${this.props.source}.query`, {query}),
-    200
-  );
+  saveQueryMetrics = debounce(query => {
+    if (query) {
```

Comment:
style: prefer early returns so that we avoid nesting

e.g. `if (!query) return;`